### DISCUSSION TITLE
Version 1.2.12

### DISCUSSION THREAD

**Discussion by bengtmartensson** [bengtmartensson/IrpTransmogrifier#225] [General]
Medium-importance bug fixed, #224.

Will probably make a minor release ("1.2.12", Milestone 12) in the (somewhat) near future.

**Comment by seanyoung** [comment]
FWIW I think I will be opening more issues/pull request in the near future. I'm currently comparing the IrpTransmogrifier IRP encoder against the rust [irp crate](https://crates.io/crates/irp) using [fuzzing](https://github.com/seanyoung/cir/blob/main/irp/encoder_fuzzer/src/main.rs). There are quite a few issues to work through, but some of those look like IrpTransmogrifier is behaving strangely. These are all strange edge cases so nothing fundamental.

**Comment by bengtmartensson** [comment]
Planned release date for 1.2.12 is March 1 (Milestone 12).

@seanyoung : anything more?

**Reply by seanyoung** [reply]
I do have some more stuff, I am slowly working through the fuzzing results.

**Reply by seanyoung** [reply]
Actually, I checked and I don't have anything right now. I *may* have more as I work through the results. I really don't mind when you do a release.

**Comment by bengtmartensson** [comment]
Current snapshot (commit 1d5a12808e3224ec7f6e2d38ac9d834c44d65006) is considered "1.2.12 release candidate 1".  (No, I do not plan to tag it.) Any "release candidate feedback" please ASAP.

**Comment by bengtmartensson** [comment]
Released. Milestone 12 closed.
